admin/src/templates/display_privacy_policies.html

```diff
@@ -0,0 +1,65 @@
+<div class="loader" ng-show="loading"></div>
+<div class="section privacy-policies" ng-hide="loading">
+  <div class="container-fluid">
+    <form id="privacy-policy-upload">
+      <div class="row selection-heading">
+        <div class="col-xs-4">
+          <span translate>Language</span>
+        </div>
+        <div class="col-xs-4">
+          <span translate>display.privacy.policies.current</span>
+        </div>
+        <div class="col-xs-4 text-right">
+          <span translate>display.privacy.policies.update</span>
+        </div>
+      </div>
+      <div class="row selection" ng-repeat="language in languages track by language.code">
```

Comment:
Currently the rows look like they're clickable because they highlight on hover but clicking them doesn't do anything.
```suggestion
      <div class="row" ng-repeat="language in languages track by language.code">
```

Reply:
Done.